docs/getting-started/development-environment.md

```diff
@@ -53,6 +53,12 @@ The development server does not include any checks by default, but they can be
 enabled using the `--check` flag. Note that using the flag may consume more
 system resources and slow things down.
 
+```bash
+
+$ yarn --check
```

Comment:
Did you try this out? 😅

Reply:
For some strange reason it's not working for me after fetching new merges  from upstream branch,I'll rectify it and create new pull request .

Sorry for the  inconvenience caused by me,  will never happen again.

Reply:
Np, you can update this PR too!

I think this is a good clarification for the docs, just gotta have the correct command 😅 
Best remove some of the padding inside the codeblock while at it too, since it takes up a bit much space

Reply:
cool....will do it